docs/pages/blog/mui-x-end-v6-features.md

```diff
@@ -152,39 +153,45 @@ To illustrate some of these improvements, the following table display a benchmar
   </tr>
 </table>
 
-As we can see, the current version operates 7.9 times faster; the average time has dropped from 400.7 milliseconds in the previous version to just 50.4 milliseconds in the current one.
+As we can see, the current version operates 7.9 times faster;
+the average time has dropped from 400.7 milliseconds in the previous version to just 50.4 milliseconds in the current one.
 
 :::info
-Note that these results are based on raw filtering speed, and not end-to-end operation (which includes user typing, filtering and rerender).
+Note that these results are based on raw filtering speed and not end-to-end operation (which includes user typing, filtering, and rerender).
 :::
 
 ## What's next?
 
 We're now working on the next MUI X major, v7, which will include many new features and improvements.
-
 Most notably:
 
-- DateTime and TimeRangePicker
-- Improved accessibility on date and time fields
-- New UI for column management for the [`DataGrid`](/x/react-data-grid/)
-- Pivoting for the [`DataGridPremium`](/x/react-data-grid/#premium-plan)
+- [Date and Time pickers](/x/react-date-pickers/) iteration:
+  - New component: `DateTime` and `TimeRangePicker`
```

Comment:
```suggestion
  - New componenst: `DateTime` and `TimeRangePicker`
```

I wondered if we were about to add these new components entirely or do something specific with them?
It wasn't clear.